explorer_backend/services/sqlite.ts

```diff
@@ -10,26 +10,58 @@ db.exec(`
 CREATE TABLE IF NOT EXISTS code (
     created_at TIMESTAMP,
     hash TEXT PRIMARY KEY,
-    code TEXT
+    code TEXT,
+    content TEXT
 );
 `);
 
-const getStmt = db.prepare("SELECT code FROM code WHERE hash = ?");
+const codeCnt = db.get(`SELECT COUNT(*) AS cnt
+FROM pragma_table_info('code')
+WHERE name = 'content';`);
+console.log(codeCnt);
+if (codeCnt.cnt == 0) {
+db.exec(`alter table code add column content TEXT`);
+}
 
-export const getCode = (hash: string): string | null => {
-  const result = getStmt.get(hash) as { code: string } | undefined;
-  return result?.code || null;
+db.exec(`
+  UPDATE code 
+  SET content = json_object('Code.sol', code) 
+  WHERE content IS NULL;
+  `
+);
+
+const getStmt = db.prepare(`
+  SELECT content
+  FROM code
```

Comment:
i don't like the names `code` and `content` particularly. Both are code essentially as far as i see.